bin/jurassic-tube-setup.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Exit if any command fails.
+set -e
+
+echo "Checking if ${PWD}/docker/bin/jt directory exists..."
+
+if [ -d "${PWD}/docker/bin/jt" ]; then
+    echo "${PWD}/docker/bin/jt already exists."
+else
+    echo "Creating ${PWD}/docker/bin/jt directory..."
+    mkdir "${PWD}/docker/bin/jt"
+fi
+
+echo "Downloading the latest version of the installer script..."
+echo
+
+# Download the installer (if it's not already present):
+if [ ! -f "${PWD}/docker/bin/jt/installer.sh" ]; then
+    # Download the installer script:
+    curl "https://jurassic.tube/get-installer.php?env=wcpay" -o ${PWD}/docker/bin/jt/installer.sh && chmod +x ${PWD}/docker/bin/jt/installer.sh
```

Comment:
So the only foible I have with this, is over here I'm using `env=wcpay` to get the setup installer for Jurassic Tube, where really we should probably be adding `wcship` as an additional case to the switch over [here](https://github.com/Automattic/jurassic-tube/blob/master/get-installer.php#L7) and then passing `env=wcship` instead. However, while looking a bit neater, ultimately this would make everything work just the same.